OpenUpSA/wazimap-ng-ui
# Refresh state when ever we fetch indicator data while loading data mapper

## Description
This issue was because state for loading was not getting refreshed
So I applied 2 solutions:
* Rather than passing indicatorData specifically to children, I started using it from parent. so Data will loaded when we will use it
* Rather than setting loading to True/False we are setting counter and incrementing it whenever we get response from api.
This ensures that we always refresh state when we get response

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-728

## How is it tested automatically?
No automated tests for this one

## How should a reviewer test it locally
There is no definite way to recreate this issue
* Select a subindicator in datamapper
* visit child geo
* data mapper menu will be refreshed
* click any of the subindicator from same indicator and see if works correctlly

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
